tests/src/Traits/EntityTrait.php

```diff
@@ -75,10 +75,22 @@ protected static function getEntityByLabel(string $entity_type_id, string $label
    */
   protected static function entityTypeAliases(): array {
     return [
+      'asset_distribution' => 'rdf_entity',
+      'asset_release' => 'rdf_entity',
+      'contact_information' => 'rdf_entity',
```

Comment:
This method should take human readable aliases, so there should not be any underscores here.